### PR TITLE
feat(route/inoreader): add podcast support

### PR DESCRIPTION
**Pull request by IvanWng97**
## 该 PR 相关 Issue / Involved Issue

Close #

## 路由地址示例 / Example for the Proposed Route(s)

```routes
/inoreader/rss/1005137674/user-favorites
```

## 新 RSS 路由检查表 / New RSS Route Checklist
  
- [ ] 新的路由 New Route
  - [ ] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [ ] 文档说明 Documentation
  - [ ] 中文文档 CN
  - [ ] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/inoreader/rss.js [hunk 1]
```diff
@@ -1,18 +1,39 @@
...
     const feed = await parser.parseURL(rssUrl);
-    feed.items = feed.items.map((item) => ({
-        title: item.title,
-        pubDate: item.pubDate,
-        link: item.link,
-        description: item.content,
-        category: item.categories,
-    }));
+    feed.items = feed.items.map((item) => {
+        if (item?.enclosure?.type.includes('audio')) {
```
Comment (TonyRL) [inline lib/v2/inoreader/rss.js:11]: https://deepscan.io/dashboard/#view=project&tid=6244&pid=8135&bid=92448&subview=pull-request&prid=1583026

**Review by TonyRL — COMMENTED** [review]
You need to set `ctx.state.data.itunes_author` and `ctx.state.data.image` for podcast feed

**Comment by NavePnow** [comment]
> You need to set `ctx.state.data.itunes_author` and `ctx.state.data.image` for podcast feed

this route may contain the mixture of feeds. some of them are podcast supported, some of them aren't. Besides, these podcast feeds maybe not from the same author. how can i deal with that.

**Review by TonyRL — COMMENTED** [review]
Please take a look at 
-   [A Podcaster’s Guide to RSS](https://help.apple.com/itc/podcasts_connect/#/itcb54353390)
-   [RSS feed guidelines for Google Podcasts](https://support.google.com/podcast-publishers/answer/9889544)

It wouldn't be recognized as a podcast feed without `itunes_author` and `image`.

**Pull request merged**